### PR TITLE
fix(components): consistent size for checkbox and radio

### PR DESCRIPTION
**Pull request by olaf-k**
The current Figma file makes the checkbox and the radio controls 20×20px with a 10px gap from the label.
I can either update [my current PR for the radio](https://github.com/Vonage/vivid-3/pull/696) and align it with the checkbox or merge this to change the checkbox.

### CODE REVIEW

File: libs/components/src/lib/checkbox/checkbox.scss [hunk 1]
```diff
@@ -67,9 +72,13 @@ $gutter: 4px;
 }
 
 label {
-	color: var(#{constants.$vvd-color-canvas-text});
+	color: var(#{appearance.get-appearance-token(text)});
```
Comment (olaf-k) [inline libs/components/src/lib/checkbox/checkbox.scss:75]: see https://github.com/Vonage/vivid-3/pull/696#discussion_r983487597

Comment (rachelbt) [reply libs/components/src/lib/checkbox/checkbox.scss:75]: this causes the text to be white when the chackbox is checked:

**Review by rachelbt — COMMENTED** [review]
you are missing ui test png's :)

File: libs/components/src/lib/checkbox/checkbox.scss (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/checkbox/checkbox.scss:75]: this PR should scope the consistence in sizing. I created another PR to modify checkbox styling Vonage/vivid-3#705

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**